Makefile

```diff
@@ -0,0 +1,59 @@
+##@ Makefile
+
+##@ Define the default version
+VERSION ?= latest
+
+tools.airbyte-ci.install: ## Install airbyte-ci
```

Comment:
Another thing to keep in mind is that @postamar worked on [an auto install of airbyte ci from gradle ](https://github.com/airbytehq/airbyte/blob/master/build.gradle#L448).
We should not break it with this PR. I don't know if the best approach would be to make gradle call the `make command` or to call a bash script.

Reply:
I dont mind the idea of moving it to a separate file. But I would like to keep the entry point for a "user upgrading/installing airbyte-ci" and "airbyte-ci upgrading/installing airbyte-ci" the same. Doing so helps limit regressions since theres 1 instead of 2 external interfaces to think about.

Reply:
Also @alafanechere whats the best way to trigger this area of the gradle code that installs airbyte-ci?

I think it should still work but want to be sure

Reply:
I wouldn't worry about the gradle tasks, it's fine to leave them as-is. In fact, this may be more desirable.